pyproject.toml

```diff
@@ -1,20 +1,58 @@
-[tool.ruff]
-line-length = 88
+[tool.coverage.run]
+branch = true
+parallel = true
+source = ["."]
+relative_files = true
 
-select = ["E", "F", "I"]
-ignore = [ "E501" ]
+[tool.coverage.html]
+show_contexts = true
 
+[tool.coverage.report]
```

Comment:
Only our test code is using python, so I don't think we need code coverage for this.